code/index.html

```diff
@@ -3,45 +3,61 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project Name</title>
+    <title>Project 5</title>
     <link rel="stylesheet" href="style.css" />
     <link
       href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
       rel="stylesheet"
     />
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link 
+    href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@500&display=swap" 
+    rel="stylesheet"
+    />
   </head>
   <body>
     <aside class="question-section">
-      <button id="restart" class="outlined-button">RESTART</button>
-
+  
+      <button id="restart" class="buttonRestart">RESTART</button>
```

Comment:
I would recommend naming classes with snake-case. This is no affecting the code, but it's considered a standard way to name classes.

Reply:
Yes, this is Technigo's naming convention so all CSS selectors should be written in kebab-case. 🍡